platform/wm/rhodes/MainWindow.cpp

```diff
@@ -1241,8 +1241,11 @@ LRESULT CMainWindow::OnExecuteJSCommand(WORD /*wNotifyCode*/, WORD /*wID*/, HWND
 {
     TNavigateData* nd = (TNavigateData*)hWndCtl;
     if (nd) {
-        LPTSTR wcurl = (LPTSTR)(nd->url);
-        if (wcurl) 
+        wchar_t wcurl[2046];
+	memset(wcurl,'0',2046);
```

Comment:
Do you mean to fill the buffer with the character '0'? Shouldn't it be memset(wcurl,0,2046)? And wchar_t is a wide character type (2 bytes), so the buffer will actually be 2 * 2046 bytes long.